package.json

```diff
@@ -60,11 +60,10 @@
     "format-js": "eslint lib test --fix",
     "check:lint": "eslint lib test",
     "precommit": "check-clang-format",
-    "test": "mocha test",
     "docs": "jsdoc2md --template etc/README.hbs --plugin dmd-clear --files lib/kerberos.js > README.md",
-    "rebuild": "prebuild --runtime napi --compile",
-    "prebuild": "prebuild --runtime napi --strip --verbose --all",
-    "release": "standard-version -i HISTORY.md"
+    "test": "mocha test",
+    "prepare": "",
```

Comment:
What's the reason for the noop here?

Reply:
Happy to remove this, but in porting over the script line 81 we run prepare, and it fails because we don't have a prepare step. I can remove the `npm run prepare` step making this package slightly different than our others or we have a noop prepare so that it matches the rest.

Maybe this is silly 😅 and we can just add prepare in when we actually have something to do for this step

Reply:
I removed prepare with a comment to add it back with TS

Reply:
Great, thanks!